docs/api/api.py

```diff
@@ -113,3 +113,6 @@ def helm(pathToChartDir: Union[str, LocalPath]) -> Blob:
 
 def fail(msg: str) -> None:
   """Raises an error that cannot be intercepted. Can be used anywhere in a Tiltfile."""
+
+def blob(input: str) -> Blob:
```

Comment:
i'm still concerned that `blob` is a bad name for this function, but not sure i can come up with a better one